packages/dashboard/src/pages/actions/[id].tsx

```diff
@@ -415,7 +415,7 @@ export default function Index() {
 									return router.push("/actions");
 								}}
 								className={
-									"flex w-full justify-center rounded border border-neutral-300 bg-white px-6 py-2 text-base font-medium text-neutral-800 focus:outline-none focus:ring-2 focus:ring-neutral-800 focus:ring-offset-2 sm:mt-0 sm:w-auto sm:text-sm"
+									"flex w-fit justify-center rounded border border-neutral-300 bg-white px-6 py-2 text-base font-medium text-neutral-800 focus:outline-none focus:ring-2 focus:ring-neutral-800 focus:ring-offset-2 sm:mt-0 sm:w-auto sm:text-sm"
```

Comment:
Cancel button can go too wide in certain screen size making it looks weird. I applied the same treatment for other cancel buttons.

Before:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/29f7b707-fc5a-4e81-8cc8-237c2f381739" />

<img width="1475" alt="image" src="https://github.com/user-attachments/assets/821e44a5-0b36-4f46-98e9-db2c02b7b17f" />

After:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/68d95923-3870-4521-b463-3fafc5e19bd2" />

<img width="1480" alt="image" src="https://github.com/user-attachments/assets/17a207b0-d7b1-4f15-920c-601600298eda" />